src/analysis/retail/evoker/augmentation/modules/talents/BlisteringScales.tsx

```diff
@@ -0,0 +1,161 @@
+import SPELLS from 'common/SPELLS/evoker';
+import TALENTS from 'common/TALENTS/evoker';
+import { formatNumber } from 'common/format';
+
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import ItemDamageDone from 'parser/ui/ItemDamageDone';
+import Events, { ApplyBuffStackEvent, CastEvent, DamageEvent } from 'parser/core/Events';
+import { calculateEffectiveDamage } from 'parser/core/EventCalculateLib';
+import { REACTIVE_HIDE_MULTIPLIER } from '../../constants';
+import { chitinBuffStackGained } from '../normalizers/CastLinkNormalizer';
+
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
+import TalentSpellText from 'parser/ui/TalentSpellText';
+import DonutChart from 'parser/ui/DonutChart';
+import { SpellLink } from 'interface';
+
+/**
+ * Blistering Scales is essentially Augmentations external
+ * it provides your target with 30% of your armor along with doing AoE
+ * to nearby enemies when said target is meleed. Consuming 1 stack.
+ * This effect has an ICD of 3-4s
```

Comment:
irrelevant but it's lower, over 26s you can end up at 2-3 stacks left (26/13 => ~2s ICD)

Reply:
will be amended